src/Common/tests/System/Collections/IEnumerable.Generic.Tests.cs

```diff
@@ -283,16 +276,14 @@ protected enum EnumerableOrder
                         "enumerator.MoveNext() returned true past the expected end.");
                 }
 
-                if (!Enumerator_Current_UndefinedOperation_Throws)
+                if (Enumerator_Current_UndefinedOperation_Throws)
```

Comment:
That's good, but we should still be testing that concurrent calls have the same behavior.